test/harness.spec.js

```diff
@@ -229,6 +229,22 @@ describe('Harness tests', () => {
       });
     });
 
+    it('set number to 0', async () => {
```

Comment:
would you mind putting the issue number or PR number in the bug title? eg.  `#128 - set false value in answer`